style.css

```diff
@@ -314,6 +314,58 @@ nav ul li span {
   padding: 6px 12px;
 }
 
+.contact-bg-color {
+  border-top-left-radius: 50px;
+  background-color: #6070ff;
+  background-image: url('./assets/images/contact-1.svg');
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: right;
+  padding: 125px 0 45px 0;
+  margin-top: 113.85px;
+}
+
+.contact-title {
+  font-size: 40px;
+  line-height: 50px;
+  text-align: center;
+  color: #fff;
+}
+
+.contact-sub-title {
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #fff;
+}
+
+.contact-form {
+  margin-top: 87px;
+}
```

Comment:
- [ ] Your design deserves a :+1:, kindly use flexbox to align the contents of your form as it is mentioned in the project requirement.

<img src="https://user-images.githubusercontent.com/17243325/205974932-161613f1-cc01-463c-af7f-b317148af5cd.png">